pkg/coremodel/playlist/coremodel.cue

```diff
@@ -0,0 +1,49 @@
+package playlist
+
+import (
+	"github.com/grafana/thema"
+)
+
+thema.#Lineage
+name: "playlist"
+seqs: [
+	{
+		schemas: [
+			{//0.0
+				// Unique numeric identifier for the playlist.
+				// TODO: Isolate or remove identifiers local to a Grafana instance
+				id?: int64 @reviewme()
+				// Unique playlist identifier. Generated on creation, either by the
+				// creator of the playlist of by the application.
+				uid?: *"" | string & len(uid)<=80 @reviewme()
+				// Name of the playlist.
+				name?: *"" | string & len(name)<=255 @reviewme()
+				// Interval
+				// TODO: Figure out the type of this, and validate that it's how long between shifts between items in the playlist.
+				interval?: *"5m" | string & len(interval)<=255 @reviewme()
+				// Playlist item
+				items?: #Items
+
+				///////////////////////////////////////
+				// Definitions (referenced above) are declared below
+
+				#Items: {
+					// Unique numeric identifier for the item references by the playlist.
+					// TODO: Isolate or remove identifiers local to a Grafana instance
+					id?: int64 @reviewme()
+					// Unique identifier for the item references by the playlist.
+					uid?: string
+					// Type of the item.
+					type?: "dashboard_by_id" | "dashboard_by_tag" | "dashboard_by_uid"
```

Comment:
I don't think "dashboard_by_uid" is a real thing yet -- at least the UI does not create one.  This seems like a good opportunity to make everything save as UID not a string version of the internal ID 🤷 .

This is what you get in main when creating a playlist today:
```
{
  "id": 2,
  "uid": "sumHSTq7k",
  "name": "second playlist",
  "interval": "5m",
  "items": [
    {
      "id": 7,
      "playlistid": 2,
      "type": "dashboard_by_id",
      "title": "General/DEBUG",
      "value": "604",
      "order": 1
    },
    {
      "id": 8,
      "playlistid": 2,
      "type": "dashboard_by_tag",
      "title": "all-panels",
      "value": "all-panels",
      "order": 2
    },
    {
      "id": 9,
      "playlistid": 2,
      "type": "dashboard_by_id",
      "title": "gdev dashboards/Bar Gauge Demo",
      "value": "1250",
      "order": 3
    }
  ]
}
```

Reply:
Ideally we can get playlist to look like:
```
{
  "uid": "sumHSTq7k",
  "name": "second playlist",
  "interval": "5m",
  "items": [
    {
      "type": "dashboard_by_id",
      "title": "General/DEBUG",
      "value": "dashboardsuid...",
    },
    {
      "type": "dashboard_by_tag",
      "title": "all-panels",
      "value": "all-panels",
    },
    {
      "type": "dashboard_by_id",
      "title": "gdev dashboards/Bar Gauge Demo",
      "value": "dashboardsuid...",
    }
  ]
}
```

If we get bold... this could even be.  IE replace the generic "value" with a more meaningful flavor of the same content
```
{
  "uid": "sumHSTq7k",
  "name": "second playlist",
  "interval": "5m",
  "items": [
    {
      "type": "dashboard_by_id",
      "title": "General/DEBUG",
      "uid": "dashboardsuid...",
    },
    {
      "type": "dashboard_by_tag",
      "title": "all-panels",
      "tags": ["all-panels"],
    },
    {
      "type": "dashboard_by_id",
      "title": "gdev dashboards/Bar Gauge Demo",
      "uid": "dashboardsuid...",
    }
  ]
}
```

Reply:
> I don't think "dashboard_by_uid" is a real thing yet

It's not, no. That was something I added 🙂 

> If we get bold... we could replace the generic "value" with a more meaningful flavor of the same content

Yeah, I was looking at doing that. I believe I'll save it for another iteration.